ROMFS/px4fmu_common/init.d-posix/rc.vars

```diff
@@ -0,0 +1,33 @@
+#!/bin/sh
+# shellcheck disable=SC2154,SC2034
+
+#global
+#PX4_SIM_MODEL=iris
+#PX4_ESTIMATOR=ekf2
+
+simulator_tcp_port=$((4560+px4_instance))
+
+# Check if PX4_SIM_HOSTNAME environment variable is empty
+# If empty check if PX4_SIM_HOST_ADDR environment variable is empty
+# If both are empty use localhost for simulator
+if [ -z "${PX4_SIM_HOSTNAME}" ]; then
+  if [ -z "${PX4_SIM_HOST_ADDR}" ]; then
+    echo "PX4 SIM HOST: localhost"
+    simulator_opts="-c $simulator_tcp_port"
+  else
+    echo "PX4 SIM HOST: $PX4_SIM_HOST_ADDR"
+    simulator_opts="-t $PX4_SIM_HOST_ADDR $simulator_tcp_port"
+  fi
+else
+  echo "PX4 SIM HOST: $PX4_SIM_HOSTNAME"
+  simulator_opts="-h $PX4_SIM_HOSTNAME $simulator_tcp_port"
+fi
+
+udp_offboard_port_local=$((14580+px4_instance))
```

Comment:
I'm not a fan of having variables set here and then used in another file. Can we keep these things together?

Reply:
Variables defined here can be used in many parts (rcS itself, rc.mavlink and so on).
Our goal is easy changing of values used in other places.

Reply:
That's also the downside of it, but at this point I don't mind too much. Which ones do you care about specifically?

Reply:
Code redone, parts:
px4-rc.params
px4-rc.simulator
px4-rc.mavlink